devtools/maven/src/test/resources/projects/simple-kogito/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.acme</groupId>
+  <artifactId>acme</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <properties>
+    <quarkus.version>@project.version@</quarkus.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <dependencies>
+  <dependency>
+      <groupId>@project.groupId@</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
+      <version>${quarkus.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>@project.groupId@</groupId>
+      <artifactId>quarkus-arc</artifactId>
+      <version>${quarkus.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>@project.groupId@</groupId>
+      <artifactId>quarkus-jsonp</artifactId>
+      <version>${quarkus.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>@project.groupId@</groupId>
+      <artifactId>quarkus-kogito</artifactId>
```

Comment:
You will need to also add a test scoped dependency to quarkus-kogito to the pom for the maven plugin. Otherwise it is possible for maven to decide to build the plugin before the kogito module and the build will fail.

Longer term we want to add a more generic way of doing hot reload tests for extensions via some kind of dev mode test framework for extensions, and when we have this we will likely try and move these tests to there.

Reply:
added the dependency the maven plugin pom as requested